defaults/test_default.yaml

```diff
@@ -266,3 +266,6 @@ run_scylla_doctor: false
 nemesis_double_load_during_grow_shrink_duration: 0
 
 enterprise_disable_kms: false
+
+num_zero_token_nodes: 0
+arbitor_dc: 0
```

Comment:
I thought they got rid of this concept of arbitor_dc.

Reply:
removed obsolete config param